src/main/java/com/gtnewhorizons/gtnhgradle/UpdateableConstants.java

```diff
@@ -16,17 +16,22 @@ public class UpdateableConstants {
     // https://github.com/GTNewHorizons/ExampleMod1.7.10/releases
     public static final @NotNull String NEWEST_BLOWDRYER_TAG = "0.2.2";
 
-    /** Latest version of UniMixins */
+    /** Latest version of UniMixins for 1.7.10 */
     public static final String NEWEST_UNIMIXINS = "io.github.legacymoddingmc:unimixins:0.1.18";
 
+    /** Latest version of MixinBooter for 1.12.2 */
+    public static final String NEWEST_MIXINBOOTER = "zone.rong:mixinbooter:9.3";
+
     /** Latest version of Jabel for modern Java support */
     public static final @NotNull String NEWEST_JABEL = "com.github.bsideup.jabel:jabel-javac-plugin:1.0.1";
     /** Latest version of GTNHLib for modern Java support */
     // https://github.com/GTNewHorizons/GTNHLib/releases
     public static final @NotNull String NEWEST_GTNH_LIB = "com.github.GTNewHorizons:GTNHLib:0.5.18";
     /** Latest version of GTNHLib for modern Java support */
     // https://github.com/GTNewHorizons/lwjgl3ify/releases
-    public static final @NotNull String NEWEST_LWJGL3IFY = "com.github.GTNewHorizons:lwjgl3ify:2.1.4";
+    public static final @NotNull String NEWEST_LWJGL3IFY_1710 = "com.github.GTNewHorizons:lwjgl3ify:2.1.4";
```

Comment:
Don't break the ABI here by renaming the constant, just add a new _1122 variant while keeping 1.7.10 unlabelled